src/templates/ship-template.json

```diff
@@ -2,7 +2,7 @@
   "label": "weather/surface-based-observations/ship-hourly",
   "title": "Hourly, or lower frequency, synoptic weather observations from ships ($CENTRE_ID)",
   "identifier": "urn:wmo:md:$CENTRE_ID:$DATA_POLICY.surface-based-observations.ship-hourly",
```

Comment:
Can/should we rename to:

`"identifier": "urn:wmo:md:$CENTRE_ID:$DATA_POLICY.surface-based-observations.ship"`

...for consistency?